pkg/plugins/virtual/virtual_plugin.go

```diff
@@ -122,7 +119,62 @@ func (p *VirtualPlugin) IsSystemService() bool {
 func needVfioDriver(state *sriovnetworkv1.SriovNetworkNodeState) bool {
 	for _, iface := range state.Spec.Interfaces {
 		for i := range iface.VfGroups {
-			if iface.VfGroups[i].DeviceType == constants.DeviceTypeVfioPci {
+			if iface.VfGroups[i].DeviceType == consts.DeviceTypeVfioPci {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+// SyncNodeStateVirtual attempt to update the node state to match the desired state in virtual platforms
+func SyncNodeStateVirtual(newState *sriovnetworkv1.SriovNetworkNodeState, helpers plugin.HostHelpersInterface) error {
```

Comment:
nit, does it need to be exported now? since it's no longer called from utils pkg?